spec/xcpretty/formatters/formatter_spec.rb

```diff
@@ -57,6 +57,11 @@ module XCPretty
 )
     end
 
+    it "formats linker warnings" do
+      @formatter.format_ld_warning("ld: embedded dylibs/frameworks only run on iOS 8 or later").should ==
+      "#{@formatter.yellow("⚠️  ld: embedded dylibs/frameworks only run on iOS 8 or later")}"
```

Comment:
Line is too long. [93/80]<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.<br>Prefer single-quoted strings inside interpolations.